tests/string_test.py

```diff
@@ -665,6 +665,9 @@ def test_encoding(self):
         self.assertTrue((a3 == a3.encode('ascii').decode('ascii')).all())
 
     def test_idna_utf16(self):
+        # skipping this test until issue is resolved
```

Comment:
could we just use the pytest skip decorator? Something like 
`@pytest.mark.skip(reason="Skipping until issue resolved.")`

Reply:
i agree, i think this would be cleaner

Reply:
Oh ya, that's way better than what I was doing, thanks for the suggestion; updated.